edgraph/access_ee_test.go

```diff
@@ -100,8 +99,10 @@ func TestGetAccessJwt(t *testing.T) {
 		{3456789012, "user3", []string{"702", "703"}},
 	}
 
+	worker.Config.AccessJwtTtl = 20 * time.Second
```

Comment:
Do you think its a good idea to override it here instead of in config?

Reply:
This is a unit test. There is not config.